src/FluffySpoon.AspNet.LetsEncrypt.Azure/AzureAppServiceSslBindingCertificatePersistenceStrategy.cs

```diff
@@ -211,6 +237,39 @@ await client.WebApps.Manager
 								sslState: SslState.SniEnabled,
 								thumbprint: azureCertificate.Thumbprint),
 							appTuple.Slot.Name);
+						}
+					}
+				}
+
+				logger.LogDebug($"Getting app settings");
+
+				var appSettings = await client.WebApps.Manager
+					.WebApps
+					.GetByResourceGroup(appTuple.App.ResourceGroupName, appTuple.App.Name)
+					.GetAppSettingsAsync();
+
+				var loadCertificatesSetting = appSettings.ContainsKey("WEBSITE_LOAD_CERTIFICATES") ? appSettings["WEBSITE_LOAD_CERTIFICATES"].Value : String.Empty;
+				var certThumbprintsToLoad = loadCertificatesSetting.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).ToList();
+				if (!certThumbprintsToLoad.Contains(azureCertificate.Thumbprint))
+				{
+					logger.LogInformation($"Adding certificate thumbprint {azureCertificate.Thumbprint} to WEBSITE_LOAD_CERTIFICATES app setting");
+
+					certThumbprintsToLoad.Add(azureCertificate.Thumbprint);
+
+					loadCertificatesSetting = String.Join(",", certThumbprintsToLoad);
+
+					try
+					{
+						await client.WebApps.Manager
+							.WebApps
+							.GetByResourceGroup(appTuple.App.ResourceGroupName, appTuple.App.Name)
+							.Update()
+							.WithAppSetting("WEBSITE_LOAD_CERTIFICATES", loadCertificatesSetting)
+							.ApplyAsync();
+					}
+					catch (Exception ex)
+					{
+						logger.LogError(ex, $"Error updating app settings for {appTuple.App.Name}");
```

Comment:
Instead of the `$` string operator, can we switch to the structured format of logging instead? It is more flexible.

Reply:
Done.